includes/documents/class-wcpdf-invoice.php

```diff
@@ -58,8 +58,11 @@ public function init() {
 
 		if ( isset( $this->settings['display_date'] ) && $this->settings['display_date'] == 'order_date' && !empty( $this->order ) ) {
 			$this->set_date( $this->order->get_date_created() );
+			$this->set_display_date( 'Order Date' );
```

Comment:
Maybe this string should be `order_date`? Then we could also have a getter method to retrieve the date label. We could also use that method to display the date options here: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/87b6ee11bde97705770300c3de724faf5ff03105/includes/documents/class-wcpdf-invoice.php#L267-L268

Reply:
Implemented suggested changes.